content/sensu-go/6.1/observability-pipeline/observe-schedule/subscriptions.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Subscriptions reference"
+linkTitle: "Subscriptions Reference"
+reference_title: "Subscriptions"
+type: "reference"
+description: "Sensu subscriptions are equivalent to topics in a traditional publish/subscribe system. With Sensu subscriptions, you can configure check requests in a one-to-many model for an entire group or subgroup of systems rather than a traditional one-to-one mapping of configured hosts or observability checks. Read this reference doc to learn how to configure subscriptions in Sensu."
+weight: 35
+version: "6.1"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-6.1:
+    parent: observe-schedule
+---
+
+Sensu uses the [publish/subscribe pattern of communication][1].
+At the core of this model are Sensu agent and check subscriptions, which are equivalent to "topics" in a traditional publish/subscribe system.
+Sensu entities become subscribers to these "topics" via their individual `subscriptions` attributes.
+Sensu checks also have a `subscriptions` attribute, where you specify which subscribers will execute the checks.
+The Sensu backend schedules checks and publishes check execution requests to entities.
```

Comment:
```suggestion
The Sensu backend schedules checks and publishes check execution requests to entities.
```
 
I think this sentence needs to better describe the connection between check subscriptions, entity subscriptions and the scheduler.



From a later passage:

> For Sensu to execute a check, the check definition must include a subscription that matches the subscription of at least one Sensu agent.

This is an important concept which isn't clear in the first paragraph.

My mental model for subscriptions is a venn diagram. We should consider adding one here or in the examples below.

Reply:
[Here's a draft Venn diagram](https://app.lucidchart.com/invitations/accept/12b80201-b9af-43a7-9e04-55335685be6f) I put together. To my eyes, this is not polished enough to go in the docs, but I am going for accuracy before beauty right now. What do you think?

Reply:
I think this is a good starting point! I have given what I hope are self-explanatory names to some of the checks you had laid out, do those changes make sense? 

Would the diagram feel more complete if the names of the subscriptions fit inside the overlapping areas? I'm not sure how to do that in a visually appealing way.